apps/firelens-stability/templates/golden-path-mountebank-fargate-with-adot-v01-11-2023/default-config.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
The rest of the template files are copied over from the main golden path fargate template. In the final PR. There will just be a modification to the main template and a new parameter called "adot_enable" which can be set to true or false to enable or disable adot.